stubs/Flask-Migrate/flask_migrate/__init__.pyi

```diff
@@ -9,20 +12,33 @@ from flask_sqlalchemy import SQLAlchemy
 _T = TypeVar("_T")
 _P = ParamSpec("_P")
 _ConfigureCallback: TypeAlias = Callable[[Config], Config]
+_AlembicConfigValue: TypeAlias = Any
 
 alembic_version: tuple[int, int, int]
 log: Logger
 
 class Config:  # should inherit from alembic.config.Config which is not possible yet
     template_directory: str | None
-    def __init__(self, *args, **kwargs) -> None: ...
+    # Same as alembic.config.Config + template_directory kwarg
+    def __init__(
+        self,
+        file_: StrPath | None = None,
+        ini_section: str = "alembic",
+        output_buffer: TextIO | None = None,
+        stdout: TextIO = sys.stdout,
+        cmd_opts: Namespace | None = None,
+        config_args: SupportsKeysAndGetItem[str, _AlembicConfigValue] | Iterable[tuple[str, _AlembicConfigValue]] = ...,
```

Comment:
`config_args` and `attributes` are used to create and update a `dict` respectively. I don't remember if it's better to type this as a `Mapping` like `alembic` did for `config_args` or maybe make a `ConvertibleToDict` type like in #10707